vpr/src/base/read_options.cpp

```diff
@@ -1950,6 +1989,11 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("off")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument<constraints_split_factor, ParseFloorplanSplit>(args.floorplan_split, "--floorplan_split")
+        .help("Used to say how many partitions the floorplan constraints file should split the blocks into.")
```

Comment:
Specifies how many partitions are created from a placement when a floorplan constraint file is generated by --write_vpr_constraints